compiler/src/utils.ml

```diff
@@ -394,6 +394,45 @@ let hierror ~loc ?funname ~kind ?sub_kind ?(internal=false) =
       raise (HiError err))
 
 
+(* -------------------------------------------------------------------- *)
+(** Splits a time in seconds into hours, minutes, seconds, and centiseconds.
+  Number of hours must be below one hundred. *)
+let hmsc f =
+  let open Float in
+  let cut f n =
+    let r = rem f n in
+    to_int r, (f -. r) /. n
+  in
+  let c, f = modf f in
+  let s, f = cut f 60. in
+  let m, f = cut f 60. in
+  let h, f = cut f 100. in
+  assert (f = 0.);
+  h, m, s, to_int (100. *. c)
+
+let pp_now =
+  let open Unix in
+  let timestamp = ref (-1.) in
+  let pp_elapsed fmt now =
+    let old = !timestamp in
+    if old >= 0. then begin
+      let diff = now -. old in
+      let h, m, s, c = hmsc diff in
+      Format.fprintf fmt "|";
+      let g = false in
```

Comment:
This two line are strange, can we replace them by let g = h > 0

Reply:
sure.

Reply:
Done.